examples/Simulations-class-GeneralSimulations.R

```diff
@@ -1,6 +1,6 @@
 data <- list(
-  Data(x = 1:3, y = 0:2, doseGrid = 1:3, ID = 1L:3L, cohort = 1L:3L),
-  Data(x = 4:6, y = 0:2, doseGrid = 4:6, ID = 1L:3L, cohort = 1L:3L)
+  Data(x = 1:3, y = 0:1, doseGrid = 1:3, ID = 1L:3L, cohort = 1L:3L),
+  Data(x = 4:6, y = 0:1, doseGrid = 4:6, ID = 1L:3L, cohort = 1L:3L)
```

Comment:
if you have 3 doses in `x` you will need 3 outcomes in `y` in each call, e.g. `y = c(0, 1, 0)`